src/react-apps/ux-editor/package.json

```diff
@@ -44,35 +44,45 @@
   "devDependencies": {
     "@types/deepmerge": "2.1.0",
     "@types/enzyme": "3.1.10",
-    "@types/jest": "23.0.0",
+    "@types/jest": "^23.3.9",
     "@types/react": "16.3.17",
     "@types/react-dom": "16.0.6",
     "@types/react-modal": "3.1.2",
     "@types/react-redux": "6.0.2",
     "@types/react-router-dom": "4.2.7",
+    "@types/react-test-renderer": "^16.0.3",
     "@types/redux": "3.6.31",
+    "@types/redux-mock-store": "^1.0.0",
     "@types/uuid": "3.4.3",
     "altinn-designsystem": "2.0.19",
     "awesome-typescript-loader": "5.0.0",
+    "babel-jest": "^23.6.0",
     "babel-preset-react": "6.24.1",
     "cross-env": "5.1.6",
     "css-loader": "0.28.11",
     "enzyme": "3.3.0",
+    "enzyme-adapter-react-16": "^1.6.0",
     "eslint": "^5.4.0",
     "eslint-config-airbnb-base": "^12.1.0",
     "html-loader": "0.5.5",
     "html-webpack-plugin": "3.2.0",
     "jest": "23.1.0",
+    "jest-environment-enzyme": "^7.0.0",
+    "jest-enzyme": "^7.0.0",
     "mini-css-extract-plugin": "0.4.0",
     "node-sass": "^4.9.3",
     "prettier": "1.13.5",
     "prettier-tslint": "^0.4.0",
+    "react-addons-test-utils": "^15.6.2",
+    "react-test-renderer": "^16.4.0",
+    "redux-mock-store": "^1.5.3",
     "resolve-url-loader": "^2.3.0",
     "rimraf": "2.6.2",
     "sass-loader": "^7.1.0",
     "source-map-loader": "0.2.3",
     "style-loader": "^0.21.0",
     "svg-inline-loader": "^0.8.0",
+    "ts-jest": "^23.10.4",
```

Comment:
Add a newer version of typescript + @types/jest to prevent syntax errors ?

Reply:
Added newer version of Typescript, but that did not fix the problem. But with an import in the "setupTests.js" file fixed it. (For me).